awx/main/migrations/0118_v400_invalid_playbooks.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.11 on 2020-06-04 03:55
```

Comment:
This migration is out of date and needs to be regenerated to point at the latest.